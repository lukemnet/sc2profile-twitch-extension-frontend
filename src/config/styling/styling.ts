const dimensions = {
  headerHeight: '123px',
  tabSectionHeight: '20px',
};

const fonts = {
  main: 'Lato',
  fallback: 'Arial, Helvetica',
  defaultFamily: 'sans-serif',
};

const styling = {
  headerHeight: dimensions.headerHeight,
  tabSectionHeight: dimensions.tabSectionHeight,
  edgePadding: '0.75rem',
  detailsHeight: `calc(100vh - ${dimensions.headerHeight})`,
  tabContainerHeight: `calc(100vh - ${dimensions.headerHeight} - ${dimensions.tabSectionHeight})`,
  transitionDuration: '150ms',
  fontFamily: `${fonts.main}, ${fonts.fallback}, ${fonts.defaultFamily}`,
};

export default styling;
