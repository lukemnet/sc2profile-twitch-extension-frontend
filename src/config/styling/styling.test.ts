import styling from './styling';

it('exports styling object', () => {
  expect(styling).toMatchSnapshot();
});
