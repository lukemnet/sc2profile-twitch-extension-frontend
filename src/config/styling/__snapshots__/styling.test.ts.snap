// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports styling object 1`] = `
Object {
  "detailsHeight": "calc(100vh - 123px)",
  "edgePadding": "0.75rem",
  "fontFamily": "Lato, Arial, Helvetica, sans-serif",
  "headerHeight": "123px",
  "tabContainerHeight": "calc(100vh - 123px - 20px)",
  "tabSectionHeight": "20px",
  "transitionDuration": "150ms",
}
`;
